kubernetes/zenko/charts/burry/templates/cronjob.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ template "burry.fullname" . }}
+  labels:
+    app: {{ template "burry.name" . }}
+    chart: {{ template "burry.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+    checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
```

Comment:
This is unnecessary for CronJobs since every time they get triggered, they will have the latest version of the ConfigMap.

Reply:
Done.